app/src/main/res/values/colors.xml

```diff
@@ -27,6 +27,7 @@
   <color name="oppiaProgressChapterFinished">#E76F51</color>
   <color name="oppiaProgressChapterNotFinished">#999999</color>
   <color name="oppiaDisabledButtonBackground">#61999999</color>
+  <color name="oppiaTextShadow">#0000001F</color>
```

Comment:
Please check this https://github.com/oppia/oppia-android/wiki/Working-on-UI#color-code for 8-character color code from mocks.

Reply:
@rt4914  changed colors 
Thankyou